### DISCUSSION TITLE
Error Kata 6

### DISCUSSION THREAD

**Discussion by Alemty** [LaunchX-InnovaccionVirtual/CursoIntroPython#301] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/99059511/154608957-020f83c1-7936-470f-b5cf-8697f36d6baa.png)

 Alguien sabe por que cuando uso el comando input se queda cargando pero nunca lo compila? 
 Ya intenté cambiando toda la linea y probando el print(planetas) y no se asigna :(

**Comment by OscarPedroza** [comment, marked as answer]
Hola, en la parte superior de Visual Studio Code te debe aparecer un cuadro de texto para ingresar la información, ingresas el texto ahi y das enter para guardar la información.

![image](https://user-images.githubusercontent.com/83786890/154611721-893d8aa5-a4da-4fbe-af82-0b1945d46490.png)

**Reply by Alemty** [reply]
Muchas gracias, en efecto era eso, como no había manejado el cuadro de texto pero en Python no me daba error, me confundí. Gracias.
